.github/workflows/lbox-develop.yml

```diff
@@ -2,9 +2,9 @@ name: LBox Develop
 
 on:
   push:
-    branches: [develop]
+    branches: [develop, v6]
```

Comment:
Add the PR here just so tests can run we can remove once V6 is merged onto develop